test/api.js

```diff
@@ -1022,3 +1022,49 @@ function generateTests(prefix, apiCreator) {
 		]);
 	});
 }
+
+function generatePassDebugTests(execArgv, expectedInspectIndex) {
+	test('pass ' + execArgv.join(' ') + ' to fork', function (t) {
+		t.plan(expectedInspectIndex === -1 ? 3 : 2);
+
+		var api = new Api({testOnlyExecArgv: execArgv});
```

Comment:
> do you mean that we should replace process.execArgv during test?

Yes.

> I think, it is not acceptable to touch global state.

True, but tests are special.

Something I've done in other projects is to have a `process` module which exports the global `process`, and then stubbing that through Proxyquire. But let's not do that unless we really have to.